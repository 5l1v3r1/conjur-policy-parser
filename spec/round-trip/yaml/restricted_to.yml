- !host
  id: app
  restricted_to: 192.168.0.0

- !user
  id: alice
  restricted_to: 192.168.0.0/16

- !user
  id: bob
  restricted_to: [ 192.168.0.1/16, 192.168.0.2/16 ]
