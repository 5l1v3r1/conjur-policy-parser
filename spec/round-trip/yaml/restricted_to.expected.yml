- !host
  id: app
  restricted_to: 192.168.0.0

- !user
  id: alice
  restricted_to: 192.168.0.0/16
